software/speller/data_collection_platform/dcp/api.py

```diff
@@ -4,7 +4,7 @@
 import os
 
 import dcp.mp.shared as shared
-from dcp import db, celery
+# from dcp import db, celery
```

Comment:
are we moving the db interaction to another section of the flask app?

Reply:
no, this is temporarily commented out to be able to run the app, but communicating directly with the db inside of `api.py` should be fine.